server/base/src/main/java/org/apache/accumulo/server/util/CheckForMetadataProblems.java

```diff
@@ -153,37 +165,45 @@ private static void checkMetadataAndRootTableEntries(String tableNameToCheck, Se
           justLoc = false;
         } else if (colf.equals(CurrentLocationColumnFamily.NAME)) {
           if (justLoc) {
-            System.out.println("Problem at key " + entry.getKey());
+            printProblemMethod.accept("Problem at key " + entry.getKey());
             sawProblems = true;
           }
           justLoc = true;
         }
       }
 
       if (count == 0) {
-        System.err
-            .println("ERROR : table " + tableNameToCheck + " (" + tableCheckId + ") is empty");
+        printProblemMethod.accept(
+            String.format("ERROR : table %s (%s) is empty", tableNameToCheck, tableCheckId));
```

Comment:
this changes `System.err` -> `System.out` when this is run from `main()` which, as far as I can tell, should be ok. To keep the functionality of `System.err`, would have to introduce yet another param which seemed a bit much for just this one print